admin/app/routes/authenticated/users/list.js

```diff
@@ -7,17 +7,19 @@ export default class ListRoute extends Route {
   queryParams = {
     pageNumber: { refreshModel: true },
     pageSize: { refreshModel: true },
+    id: { refreshModel: true },
     firstName: { refreshModel: true },
     lastName: { refreshModel: true },
     email: { refreshModel: true },
     username: { refreshModel: true },
   };
 
   async model(params) {
-    if (!params.firstName && !params.lastName && !params.email && !params.username) return [];
+    if (!params.id && !params.firstName && !params.lastName && !params.email && !params.username) return [];
 
     return this.store.query('user', {
       filter: {
+        id: params.id ? params.id.trim() : '',
```

Comment:
Cette partie ne gère pas la logique métier attendu.
- trim avant ✅ 
- trim entre ❌ 
- trim après ✅

Reply:
```suggestion
        id: params.id ? params.id.replace(/ /g, '') : '',
```
Avec cette expression régulière dans le replace tu retires tous les espaces.

Reply:
Merci 🙏🏼

Reply:
Merci 🙏🏼